types/parse/index.d.ts

```diff
@@ -422,6 +432,7 @@ namespace Parse {
         getACL(): ACL | undefined;
         has(attr: Extract<keyof T, string>): boolean;
         increment(attr: Extract<keyof T, string>, amount?: number): this | false;
+        decrement(attr: Extract<keyof T, string>, amount?: number): this | boolean;
```

Comment:
as with the one above, this will never return `true`

```suggestion
        decrement(attr: Extract<keyof T, string>, amount?: number): this | false;
```